### PR TITLE
Add a tutorial of simple classification data

### LINKED ISSUE #1048: [DOCS][CV] Add an example of using classification/simple.py to get quick insights from deepchecks
**Is your feature request related to a problem? Please describe.**
Following #1005, there needs to be an example of using the methods supplied in classification/simple.py to run some checks and get some value without defining a single VisionData object

**Describe the solution you'd like**
A simple classification use case, when some issue (such as drift or leakage) is uncovered by running a suite over a SimpleClassificationData object.

### PR DESCRIPTION
**Pull request by noamzbr**
#### Reference Issues/PRs

Closes #1048 

#### What does this implement/fix? Explain your changes.
A simple example of using the load_data method from the simple_classification_data module to quickly run checks over the dataset before even starting modeling.

### CODE REVIEW

**Comment by noamzbr** [comment]
Attached is the resulting suite
[output.zip](https://github.com/deepchecks/deepchecks/files/8358063/output.zip)

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+"""
+================================================
+Using Deepchecks Vision With a Few Lines of Code
+================================================
+
+Deepchecks Vision is built to enable validate your data and model, however complex your model and data may be. That
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:6]:
```suggestion
Deepchecks Vision is built to validate your data and model, however complex your model and data may be. That
```

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+being said, sometime there is no need to write a full-blown
+:doc:`**ClassificationData** </tutorials/vision/classification_tutorial>` or
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:8]:
```suggestion
`**ClassificationData** </tutorials/vision/classification_tutorial>` or
```

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+:doc:`**DetectionData** </tutorials/vision/detection_tutorial>`. In the case of a simple classification task,
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:9]:
```suggestion
`**DetectionData** </tutorials/vision/detection_tutorial>`. In the case of a simple classification task,
```

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+#             - test/
+#                 - class1/
+#                     image1.jpeg
+
+from deepchecks.vision.simple_classification_data import load_dataset
+
+train_ds = load_dataset('hymenoptera_data', train=True, object_type='VisionData', image_extension='jpg')
+test_ds = load_dataset('hymenoptera_data', train=False, object_type='VisionData', image_extension='jpg')
+
+# Running Deepchecks' full suite
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:57]: Shouldn't it be a regular title? (=============)

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+# That's it, we have just defined the classification data object and are ready to run the train_test_validation suite:
+
+from deepchecks.vision.suites import train_test_validation
+
+suite = train_test_validation()
+result = suite.run(train_ds, test_ds)
+
+#%%
+# Observing the results:
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:68]: Same here?

**Review by ItayGabbay — CHANGES_REQUESTED** [review]

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 6]
```diff
@@ -0,0 +1,76 @@
+"""
+================================================
+Using Deepchecks Vision With a Few Lines of Code
+================================================
+
+Deepchecks Vision is built to validate your data and model, however complex your model and data may be. That
+being said, sometime there is no need to write a full-blown
+`**ClassificationData** </tutorials/vision/classification_tutorial>` or
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:8]:
```suggestion
`**ClassificationData** </tutorials/vision/examples/plot_classification_tutorial.html>`__ or
```

File: docs/source/tutorials/vision/plot_simple_classification_tutorial.py [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+`**DetectionData** </tutorials/vision/detection_tutorial>`. In the case of a simple classification task,
```
Comment (ItayGabbay) [inline docs/source/tutorials/vision/plot_simple_classification_tutorial.py:9]:
```suggestion
`**DetectionData** </tutorials/vision/examples/plot_detection_tutorial.html>`. In the case of a simple classification task,
```

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**